src/librustc_mir/const_eval.rs

```diff
@@ -336,6 +336,20 @@ impl<'mir, 'tcx> interpret::Machine<'mir, 'tcx> for CompileTimeInterpreter<'mir,
         _unwind: Option<mir::BasicBlock> // unwinding is not supported in consts
     ) -> InterpResult<'tcx, Option<&'mir mir::Body<'tcx>>> {
         debug!("eval_fn_call: {:?}", instance);
```

Comment:
This comment is outdated btw (spotted it a few days ago and forgot about it) - also the method name isn't too clear on the fact that it can call the function, not just look for it.

Reply:
Some suggestions:

* process_fn
* eval_or_fetch_fn
* eval_fn_or_fetch_mir

Reply:
I'd loke to add `find_mir_or_eval` to the list.